docs/source/custom_resolvers.rst

```diff
@@ -36,9 +36,9 @@ The example below creates a resolver that adds 10 to the given value.
     >>> c.key
     1000
 
-Custom resolvers support variadic argument lists in the form of a comma separated list of zero or more values.
-Whitespaces are stripped from both ends of each value ("foo,bar" is the same as "foo, bar ").
-You can use literal commas and spaces anywhere by escaping (:code:`\,` and :code:`\ `), or
+Custom resolvers support variadic argument lists in the form of a comma-separated list of zero or more values.
+In a variadic argument list, whitespace is stripped from the ends of each value ("foo,bar" gives the same result as "foo, bar ").
+You can use literal commas and spaces anywhere by escaping (``\,`` and :code:`\ `), or
```

Comment:
The docs build fails if I change the :code:`\ ` to use double backticks. I think the backslash is causing an escaping problem.

Reply:
ah, ok.